### ISSUE TITLE
Creating calendar event does not work with version 1.0.0.0a14

### ISSUE THREAD

**Issue by dernstOfExtron** [microsoftgraph/msgraph-sdk-python#322]
After upgrading from 1.0.0.0a13 to 1.0.0.0a14 my app that creates events no longer works. I observe two failure modes.

1) After upgrading, my app would generate an exception when creating the desired event. My app does successfully connect to the calendar, and I can retrieve the user profile of the account.   I will attach the log output for review.

2) After creating a simple app to isolate the problem the failure does not generate an exception but does not create a proper event.  The event is at the wrong time and there is no subject.  When I run this same app using 1.0.0.0a13 the created event is complete and at the correct time.  I will attach the simple program as well.

**Comment by dernstOfExtron** [comment]
Here isthe log and simple app mentioned above.
[msgraph14a_failure.txt](https://github.com/microsoftgraph/msgraph-sdk-python/files/12476231/msgraph14a_failure.txt)
[SimpleExample.py.txt](https://github.com/microsoftgraph/msgraph-sdk-python/files/12476248/SimpleExample.py.txt)

**Comment by samwelkanda** [comment]
Hi @dernstOfExtron  

Thank you for reporting this issue. This was a bug in the way we track changes to models in our abstractions library.

Updating to `microsoft-kiota-abstractions==0.8.5` should fix the issue.
Thanks

**Comment by dernstOfExtron** [comment]
Thank you samwelkanda,

I verified that the fix works. I did have to make one change to my app.  When an event was created that did not have an attendee list, I was setting the requestBody.attendees to None (where requestBody = Event()).  This would generate an exception where earlier versions of the sdk did not.  I now only assign requestBody.attendees to an Attendee object if I have attendees. Otherwise I do not assign requestBody.attendees to anything.
